patch/driver-support-optoe-bank-support.patch

```diff
@@ -0,0 +1,307 @@
+From d654fd07f90bda8565ebbb18979dfee8d8fff3da Mon Sep 17 00:00:00 2001
+From: Wataru Ishida <wataru.ishid@gmail.com>
+Date: Wed, 2 Apr 2025 07:10:10 +0000
+Subject: [PATCH] optoe: Add CMIS Bank support for transceivers with >8 lanes
+
+This patch adds CMIS Bank support to the 'optoe3' device class in order
+to enable access to CMIS transceivers with more than 8 lanes (e.g., OSFP-XD, CPO OEs).
+
+- Bank support is enabled only for the 'optoe3' dev class.
+- The default bank size is set to 4, and can be modified via the 'bank_size' sysfs entry.
+- For 'optoe3', the 'write_max' value is updated to 2 to comply with CMIS requirements,
+  which mandate that both bank and page values be updated in a single WRITE operation.
+
```

Comment:
It’d be great if you could add a paragraph how to verify/test this to the commit message too.